### PR TITLE
s/typo/spelling

### PR DESCRIPTION
**Pull request by PhilETaylor**
### Summary of Changes

A bunch of (hopefully) un-controversial spelling changes in **comments only** for `4.0-dev`

* ok and a couple in PHP code, which make no changes other than typos and all are tested.

### CODE REVIEW

File: installation/src/Application/InstallationApplication.php [hunk 1]
```diff
@@ -142,7 +142,7 @@ public function debugLanguage()
 
 			$guesses = array();
 
-			foreach ($orphans as $key => $occurance)
+			foreach ($orphans as $key => $occurence)
```
Comment (PhilETaylor) [inline installation/src/Application/InstallationApplication.php:145]: `$occurance` is not actually used, only the `$key` part is.

File: installation/src/Controller/DisplayController.php [hunk 2]
```diff
@@ -24,14 +24,14 @@ class DisplayController extends BaseController
 	/**
 	 * Method to display a view.
 	 *
-	 * @param   boolean  $cachable   If true, the view output will be cached.
+	 * @param   boolean  $cacheable   If true, the view output will be cached.
```
Comment (HLeithner) [inline installation/src/Controller/DisplayController.php]: We use cachable in more then 140 cases in Joomla and cacheable in 24 cases on a quick search I think we should keep using cachable.

Comment (PhilETaylor) [reply installation/src/Controller/DisplayController.php]: Or we use the Joomla 4 break to fix things like this ... I'll revert this one.

Comment (PhilETaylor) [reply installation/src/Controller/DisplayController.php]: I have reverted my change.. this still leaves a 140/24 split ... that should be addressed....

**Comment by HLeithner** [comment]
The rest looks ok

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 05e1565944ac8dd49daa9f68b1cb162472cfc5d1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29808">issues.joomla.org/tracker/joomla-cms/29808</a>.</sub>

File: installation/src/Controller/DisplayController.php (hunk 2 above)
Comment (HLeithner) [reply installation/src/Controller/DisplayController.php]: Personally I don't care, at least we should use only one version. Only thing to consider is that this method is abstracted by 3rd party developer and we should check this before we make a b/c break. I don't know if we have property/option arrays with cachable in the code base.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 05e1565944ac8dd49daa9f68b1cb162472cfc5d1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29808">issues.joomla.org/tracker/joomla-cms/29808</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29808">issues.joomla.org/tracker/joomla-cms/29808</a>.</sub>

**Comment by zero-24** [comment]
Merged thanks

**Pull request merged**